cmd/single/start.go

```diff
@@ -108,13 +109,12 @@ func startAdmin(ctx context.Context, cfg Admin) error {
 
 func startPropeller(ctx context.Context, cfg Propeller) error {
 	propellerCfg := propellerConfig.GetConfig()
-	propellerScope := promutils.NewScope(propellerConfig.GetConfig().MetricsPrefix).NewSubScope("propeller").NewSubScope(propellerCfg.LimitNamespace)
-	limitNamespace := ""
-	var namespaceConfigs map[string]cache.Config
+	propellerScope := promutils.NewScope(propellerConfig.GetConfig().MetricsPrefix).NewSubScope("propeller").NewSubScope(strings.Replace(propellerCfg.LimitNamespace, ",", "-", -1))
```

Comment:
The `,` character is invalid in prometheus. Therefore I replace it with `-` here.